ProjectManager/ProjectManager/Controller/SubviewControllers/TaskListViewController.swift

```diff
@@ -0,0 +1,161 @@
+        super.init(nibName: nil, bundle: nil)
+    }
+
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    override func loadView() {
+        super.loadView()
+        view = projectListView
+    }
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        configureDataSource()
+        configureProjectListView()
+        registerLongPressedObserver()
+    }
+
+    private func configureDataSource() {
+        projectListView.dataSource = dataSource
+    }
+    
+    private func configureProjectListView() {
+        projectListView.delegate = self
+        projectListView.setHeaderText(text: type.rawValue)
+        projectListView.setHeaderItemCount(count: 0)
+        projectListView.register(cellClass: TaskCell.self, forCellReuseIdentifier: TaskCell.cellIdentifier)
+    }
+
+    func applySnapShot() {
+        var snapshot = NSDiffableDataSourceSnapshot<Section, Task>()
+        snapshot.appendSections([.main])
+        snapshot.appendItems(filteredTasks)
+        self.dataSource.apply(snapshot, animatingDifferences: true)
+    }
+    
+    func deleteTask(_ task: Task) {
+        if let index = filteredTasks.firstIndex(of: task) {
+            filteredTasks.remove(at: index)
+        }
+    }
+    
+    private func showEditProjectView(with task: Task) {
+        let editProjectViewController = EditProjectViewController(task: task)
+        editProjectViewController.editDelegate = self
+        let secondNavigationController = UINavigationController(rootViewController: editProjectViewController)
+        secondNavigationController.modalPresentationStyle = .formSheet
+        self.present(secondNavigationController, animated: true)
+    }
+}
+
+extension TaskListViewController: UITableViewDelegate {
+    func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
+        let task = filteredTasks[indexPath.row]
+        showEditProjectView(with: task)
+        tableView.deselectRow(at: indexPath, animated: true)
+    }
+    
+    func tableView(_ tableView: UITableView, trailingSwipeActionsConfigurationForRowAt indexPath: IndexPath) -> UISwipeActionsConfiguration? {
+        guard let cell = tableView.cellForRow(at: indexPath) as? TaskCell else {
+            return nil
+        }
+        let delete = UIContextualAction(style: .normal, title: Constant.deleteSwipeText) {_, _, _ in
+            if let task = cell.task {
+                self.deleteTask(task)
+            }
+        }
+        delete.backgroundColor = .red
+        return UISwipeActionsConfiguration(actions: [delete])
+    }
+}
+
+extension TaskListViewController {
+    private func registerLongPressedObserver() {
+        NotificationCenter.default.addObserver(self,
+                                       selector: #selector(showPopoverMenu),
+                                       name: Notification.Name("cellLongPressed"),
+                                       object: nil)
```

Comment:
```suggestion
        NotificationCenter.default.addObserver(
                self,
                selector: #selector(showPopoverMenu),
                name: Notification.Name("cellLongPressed"),
                object: nil
        )
```

이렇게 줄바꿈을 더 많이 하는 것 같아요. 개인 취향이 있을 수 있으니, 한 번 편한대로 골라서 사용해보세욥